Kickstarter-iOS/DataSources/ProjectPamphletContentDataSource.swift

```diff
@@ -36,7 +38,7 @@ internal final class ProjectPamphletContentDataSource: ValueCellDataSource {
       )
     }
 
-    self.set(values: [project], cellClass: ProjectPamphletMainCell.self, inSection: Section.main.rawValue)
+    self.set(values: [projectAndRefTag], cellClass: ProjectPamphletMainCell.self, inSection: Section.main.rawValue)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kickstarter/ios-oss/blob/7af5d8cc3bc516409245ef9b3c8ec9c0b49c4f0c/Kickstarter-iOS/DataSources/ProjectPamphletContentDataSource.swift#L41"></span>Line should be 110 characters or less: currently 115 characters<br />
<code>line_length</code> <code>ProjectPamphletContentDataSource.swift:41</code></td>
    </tr>
  </tbody>
</table>